usagov/vote-gov-drupal
# VOTE-2584 new block types and permissions for content managers

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2584

## Description

1. Create new Contact Identifier block type
2. Create new Partnership block type
3. Update permissions for content managers

## Deployment and testing

### Post-deploy steps

1. `lando retune`

### QA/Testing instructions

1. Log in as a Content Manager
2. Make sure you can visit http://vote-gov.lndo.site/es/admin/config/regional/translate
3. 

## Checklist for the Developer




## Checklist for the Peer Reviewers
